src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -55,6 +55,7 @@ const StyledTableToolbarContent = styled(TableToolbarContent)`
     padding-right: 1px;
     align-items: center;
     font-size: 0.875rem;
+    height: 2.25rem;
```

Comment:
```suggestion
    height: 3.125rem;
```

Reply:
@scottdickerson Can you please confirm if this is what you requested before?

Reply:
I think it looks ok as is. I can merge it